### PR TITLE
Install Tensorflow into base image

### PR DESCRIPTION
**Pull request by erikbern**
EDIT: this builds Bazel and Tensorflow from scratch. Takes like 2h on my laptop.

### CODE REVIEW

**Comment by erikbern** [comment]
Not sure how to test this though

**Comment by erikbern** [comment]
nvm `docker build . -f Dockerfile.analytics` – was pretty straightforward

i'll add something to the readme

**Comment by erikbern** [comment]
I've wasted a bunch of time on this... seems like a PITA to get it working. Something is broken with the libc version so I think we have to compile TF from source.

Any way we can just switch to Ubuntu for the analytics image?

**Comment by imsky** [comment]
we can switch, though it would make the analytics image inconsistent.

i can take a look at compiling TF - seems straightforward.

let me know what you'd like to do.

**Comment by erikbern** [comment]
Let me try this: https://github.com/tatsushid/docker-alpine-py3-tensorflow-jupyter/blob/master/Dockerfile

not straightforward but I guess someone already went through the trouble so let's see if we can use it

**Comment by imsky** [comment]
sounds good

File: Dockerfile.analytics [hunk 1]
```diff
@@ -15,5 +15,91 @@ RUN apk --no-cache --update-cache add gfortran build-base wget freetype-dev libp
...
+    && cp -p output/bazel /usr/bin/
+
+# Build Tensorflow
+ENV TENSORFLOW_VERSION 1.7.0
+RUN cd /tmp \
+    && curl -SL https://github.com/tensorflow/tensorflow/archive/v${TENSORFLOW_VERSION}.tar.gz \
+        | tar xzf - \
+    && cd tensorflow-${TENSORFLOW_VERSION} \
+    && : musl-libc does not have "secure_getenv" function \
+    && sed -i -e '/JEMALLOC_HAVE_SECURE_GETENV/d' third_party/jemalloc.BUILD \
```
Comment (imsky) [inline Dockerfile.analytics]: can you comment on the changes you had to make under L68? so that we know if we need to keep these tweaks long term etc.

**Comment by imsky** [comment]
👍

**Pull request merged**